floodfill_algorithm/3dcube/flood_fill_3d.pyx

```diff
@@ -42,8 +41,8 @@ def cyfill(np.ndarray[np.float64_t, ndim=3] data, tuple start_coords, Py_ssize_t
 
     while stack:
         x, y, z = stack.pop()
-		# set the threshold as 30 here but this could be get through an input
-        if data[x, y, z]>orig_value-thres and data[x, y, z]<orig_value+thres:
+        # set the threshold as 30 here but this could be get through an input
+        if data[x, y, z]>orig_value_float/thres and data[x, y, z]<orig_value_float*thres:
```

Comment:
This is the change to the definition of the threshold I was mentioning, we now do / and * instead of - and +